docs/content/references/components.md

```diff
@@ -76,21 +76,21 @@ The container tags follow one of two patterns:
 
 For example, if pulling from the [customer portal](https://access.crunchydata.com/), the following would both be valid tags to reference the pgBouncer container:
 
-- `ubi8-1.15-3`
-- `ubi8-1.15-5.0.3-0`
-- `centos8-1.15-3`
-- `centos8-1.15-5.0.3-0`
+- `ubi8-1.16-1`
+- `ubi8-5.1.0-0`
+- `centos8-1.16-1`
+- `centos8-5.1.0-0`
 
 The [developer portal](https://www.crunchydata.com/developers/download-postgres/containers) provides CentOS based images. For example, PgBouncer would use this tag:
 
-- `centos8-1.15-3`
+- `centos8-1.16-1`
 
 PostGIS enabled containers have both the Postgres and PostGIS software versions included. For example, Postgres 14 with PostGIS 3.1 would use the following tags:
```

Comment:
It might help (but is not necessary I think) to be explicit here about the slight change from the postgres tagging, i.e., 
postgres (lines 74-5): 
- `<baseImage>-<softwareVersion>-<buildVersion>`
- `<baseImage>-<softwareVersion>-<pgoVersion>-<buildVersion>`

postgis:
- `<baseImage>-<postgresVersion>-<postgisVersion>-<buildVersion>`
- `<baseImage>-<postgresVersion>-<postgisVersion>-<pgoVersion>-<buildVersion>`

Like I said, I think that "<softwareVersion>" covers this situation, but isn't as explicit as it could be.